recipes/pdal-parallelizer/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "pdal-parallelizer" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2c7a6e1a128450f1838e1c4cd9a71048621ceda0338a222e25510299484494ab
+
+build:
+  number: 0
+  noarch: python
+  script: "python -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - pdal-parallelizer = pdal_parallelizer.pdal_parallelizer_cli.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - click
+    - dask
+    - distributed
+    - twine
+    - pandas
+    - matplotlib-base
+    - pyproj
+    - python-pdal
+
+test:
+  imports:
+    - pdal_parallelizer  
```

```suggestion
    - pdal_parallelizer
  commands:
    - pip check
  requires:
    - pip
```
Tests some more stuff